src/lib/services/feature-toggle-service.ts

```diff
@@ -206,8 +206,12 @@ class FeatureToggleService {
         const id = await this.featureToggleStore.getProjectId(featureName);
 
         if (id !== projectId) {
-            throw new InvalidOperationError(
-                `The operation could not be completed. The feature exists, but the provided project id ("${projectId}") does not match the project that the feature belongs to ("${id}"). Try using "${id}" in the request URL instead of "${projectId}".`,
+            throw new NotFoundError(
+                `There's no feature named ${featureName} in project ${projectId} ${
+                    id === undefined
+                        ? '.'
+                        : `, but there's a feature by that name in project "${id}"`
```

Comment:
Not sure if we should provide this information in the API, but I'm just keeping the message closer to the one in the left